test/functional/testlib.bash

```diff
@@ -501,6 +501,8 @@ set_env_variables() { # swupd_function
 	export CACERT_DIR="$SWUPD_DIR/swupd_test_certificates" # trusted key store path
 	export PORT_FILE="$path/$env_name/port_file.txt" # stores web server port
 	export SERVER_PID_FILE="$path/$env_name/pid_file.txt" # stores web server pid
+	SWUPD_DIR_ABS="$(realpath "$SWUPD_DIR")"
+	export SWUPD_DIR_ABS
```

Comment:
if `SWUPD_DIR_ABS` is not dependent of a specific test environment I think it should be exported with the other independent variables at the top of the file
```
# global variables
FUNC_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
export FUNC_DIR
TEST_ROOT_DIR="$(pwd)"
export TEST_ROOT_DIR
TEST_FILENAME=$(basename "$BATS_TEST_FILENAME")
export TEST_FILENAME
export TEST_NAME=${TEST_FILENAME%.bats}
export THEME_DIRNAME="$BATS_TEST_DIRNAME"
export TEST_NAME_SHORT="$TEST_NAME"
export SWUPD_DIR="$FUNC_DIR/../.."
SWUPD_DIR_ABS="$(realpath "$SWUPD_DIR")"
export SWUPD_DIR_ABS
```